[2020/3/1 16:14:37][ERROR][uibot38385c2483da42.task][78] 运行发生错误，流程中止
错误信息如下：
uibot38385c2483da42.task 第78行： 尝试去连接一个null值 (字段 'AUTH')
